### ISSUE TITLE
Test fails with ghc 9.2.4 

### ISSUE THREAD

**Issue by alios** [fourmolu/fourmolu#215]
```
Failures:

  tests/Ormolu/CabalExtensionsSpec.hs:33:17: 
  1) Ormolu.CabalExtensions, Handle extensions from .cabal files, do not consider directories as .cabal files
       expected: Nothing
        but got: Just "/private/tmp/nix-haskell.cabal"

  To rerun use: --match "/Ormolu.CabalExtensions/Handle extensions from .cabal files/do not consider directories as .cabal files/"

Randomized with seed 1510731195
```

**Comment by brandonchinn178** [comment]
I dont think we do anything different than Ormolu here. Can you try with the ormolu repo and report the issue there instead if it fails?

**Comment by alios** [comment]
> I dont think we do anything different than Ormolu here. Can you try with the ormolu repo and report the issue there instead if it fails?

updated my initial comment ... was using nix in the first place it used v0.6 ... made it then use 0.7.0.1 and got another one .

**Comment by brandonchinn178** [comment]
Ok we can ignore the failure from 0.6, since its an old version. How are you running the tests? Can you make sure to explicitly build the executable? i.e. `stack build :fourmolu && stack test`

**Comment by alios** [comment]
I use nix to build it as I'am currently rebuilding my dev env for and with ghc-9.2.4 ... 
For now I've disabled nix from running the tests there. As soon as the build is done I will use it trying to build the current git master "on its own" (without nix) and see what happens ... maybe nix is somehow hiding the executable away when running the tests.

**Comment by brandonchinn178** [comment]
Sounds good, let me know. I could also disable integration tests by default

**Comment by brandonchinn178** [comment]
It works for me; I think the integration tests just don't work on Nix. Why does Nix run the tests? Is it important for the tests to pass on Nix? I could add an `integration-tests` flag to manually disable them

**Comment by sgillespie** [comment]
NixOS user here--I can run successfully run tests using both Stack and Cabal

**Comment by brandonchinn178** [comment]
@alios is this still an issue?
